src/main/AppWithRouter.js

```diff
@@ -7,7 +7,7 @@ import VM from "./VM"
 
 const { Header, Content } = Layout
 
-export default class App extends PureComponent {
+export default class App extends Component {
```

Comment:
This component cannot extend PureComponent because it [needs to re-render when the location changes, which is not in props or state](https://github.com/shengbeiniao/single-spa-demo/pull/1/files#diff-c5dfcc9ab7922c65ac9307fabc9b2867R29).